client/blocks/login/index.jsx

```diff
@@ -554,6 +558,9 @@ class Login extends Component {
 
 		return (
 			<div className="login__form-header-wrapper">
+				{ isGravatarLoginPage && (
+					<img src={ oauth2Client.icon } width={ 27 } height={ 27 } alt={ oauth2Client.alt } />
```

Comment:
I am not sure if `oauth2Client.alt` exists. Also you are using `oauth2Client.title` in [here](https://github.com/Automattic/wp-calypso/pull/81258#diff-53db3b94011d088eb9d347bbe08dd60fd909d858000f6f536e190ec93384bbacR145).

Reply:
Same [here](https://github.com/Automattic/wp-calypso/pull/81258/files#diff-53db3b94011d088eb9d347bbe08dd60fd909d858000f6f536e190ec93384bbacR211).